include/zephyr/bluetooth/gap.h

```diff
@@ -832,6 +832,70 @@ enum {
  */
 #define BT_GAP_PER_ADV_INTERVAL_TO_MS(interval) ((interval) * 5 / 4)
 
+/**
+ * @brief Convert milliseconds to advertising interval units (0.625 ms)
+ *
+ * @note If @p _int is not a multiple of the unit, it will round down to nearest. For example
+ * BT_GAP_MS_TO_ADV_INT(23) will become 22.5 milliseconds
+ */
+#define BT_GAP_MS_TO_ADV_INT(_int) ((uint16_t)((_int) / 0.625))
+
+/**
+ * @brief Convert milliseconds to periodic advertising interval units (1.25 ms)
+ *
+ * @note If @p _int is not a multiple of the unit, it will round down to nearest. For example
+ * BT_GAP_MS_TO_PER_ADV_INT(23) will become 22.5 milliseconds
+ */
+#define BT_GAP_MS_TO_PER_ADV_INT(_int) ((uint16_t)((_int) / 1.25))
```

Comment:
Wasn't sure whether to use `INT` or `INTERVAL`. We seem to be using both

Reply:
Personally I'd prefer `INTERVAL`, it is more consistent with the previous macro `BT_GAP_PER_ADV_INTERVAL_TO_MS`. Not going to block this because at the end of the day `INT` also works

Reply:
I prefer "interval".

Reply:
Updated to `INTERVAL`